chapters/connectors.tex

```diff
@@ -863,7 +863,7 @@ \subsection{Operators for Connection Graphs}\label{overconstrained-equation-oper
 \end{nonnormative}
 
 
-\subsection{Converting the Connection Graph into Trees and Generating Connection Equations}\label{converting-the-connection-graph-into-trees-and-generating-connection-equations}
+\subsection{Generation of Connection Graph Equations}\label{generation-of-connection-graph-equations}
```

Comment:
I think the last one would work.

Reply:
I was trying to have some similarity with the section 9.2 _Generation of Connection Equations_, and to me the suggested _Connection Graph Equations_ is meant to have the same meaning as _Equations from Connection Graphs_.  For comparison, we didn't say _Generation of Equations from Connect Equations_.

How about just changing the first one to _Connection Graphs and Their Operators_?  (This still a shorter title than the one below, so the length of it feels acceptable.)

Reply:
That is the dilemma: Have a better fitting name or a shorter one? I like *Connection Graphs and Their Operators* but *Operators for Connection Graphs* is shorter and fits fully into the menu. I would still prefer *Equations from Connection Graphs* instead of *Connection Graphs Equations* because it suggests to me more of the process. *Connection Graph Equations* suggests that it's a concept in its own right, which it is not.

Reply:
Since you say you like _Connection Graphs and Their Operators_ and this is still doesn't extend further to the right than two other headings in the same chapter, I went with this change.

Regarding the _Generation of Connection Graph Equations_, I tried to fix this by making _Connection Graph Equations_ just as much as concept as _Connection Equation_ and _Stream Connection Equations_.  What do you think of that?

Reply:
OK. Resolving.